src/frontend/packages/core/src/features/cloud-foundry/tabs/cloud-foundry-organizations/cloud-foundry-organization-spaces/cloud-foundry-space-base/cloud-foundry-space-base.component.ts

```diff
@@ -127,7 +128,27 @@ export class CloudFoundrySpaceBaseComponent implements OnDestroy {
     ).subscribe();
 
     // Add any tabs from extensions
-    this.tabLinks = this.tabLinks.concat(getTabsFromExtensions(StratosTabType.CloudFoundrySpace));
+    this.setupLinks();
+  }
+
+  private setupLinks() {
+    this.quotaLinkSub = this.cfSpaceService.space$.pipe(
+      tap((space) => {
+        let link = 'space-quota';
+
+        if (!space.entity.entity.space_quota_definition) {
+          link = 'quota';
```

Comment:
I don't think we should be linking top the org quota here, users may assume that this is the space quota.